edit-post/assets/stylesheets/main.scss

```diff
@@ -146,4 +146,4 @@ body.gutenberg-editor-page {
 		font-size: $default-font-size;
 		padding: 6px 10px;
 	}
-}
+}
```

Comment:
This file should be reverted to reintroduce the newline at the end.

Reply:
I have reverted file.